docs/core/versions/binding.md

```diff
@@ -0,0 +1,117 @@
+
+.NET Standard target frameworks are also capped in the same way. The .NET Core 2.0 SDK is capped to `netstandard2.0`.
+
+You can override the minimum runtime patch version (to higher or lower versions) in the project file, as shown in the following example:
+
+``` xml
+<RuntimeFrameworkVersion>2.0.4</RuntimeFrameworkVersion>
+```
+
+ The `RuntimeFrameworkVersion` element  overrides the default version policy. When you target multiple frameworks, you must specify the minimum runtime patch version to the specific target framework, as shown in the following example for .NET Core:
+
+``` xml
+<TargetFrameworks>netcoreapp2.0;net47</TargetFrameworks>
+<RuntimeFrameworkVersion Condition="'$(TargetFramework)' == 'netcoreapp2.0'">2.0.4</RuntimeFrameworkVersion>
+```
+
+## dotnet run rolls forward
+
+You run an application from source with [`dotnet run`](../tools/dotnet-run.md). `dotnet run` both builds and runs an application.
+
+The .NET CLI chooses the latest patch version installed on the machine. For example, if you specified `netcoreapp2.0` in your project file, and `2.0.4` is the latest .NET runtime installed, the `2.0.4` runtime is used.
+
+If no acceptable `2.0.*` version is found, a new `2.*` version will be used. For example, if you specified `netcoreapp2.0` and only `2.1.0` is installed, the application will run using the `2.1.0` runtime. This This behavior is referred to as "minor version roll-forward." Lower versions also won't be considered. When no acceptable runtime is installed, the application will not run.
+
+A few usage examples demonstrate the behavior:
+
+- 2.0.4 is required. 2.0.5 is the highest patch version available. 2.0.5 is used.
+- 2.0.4 is required. No 2.0.* versions are installed. 1.1.1 is the highest runtime installed. An error message is printed.
+- 2.0.4 is required. No 2.0.* versions are installed. 2.2.2 is the highest 2.x runtime version installed. 2.2.2 is used.
+- 2.0.4 is required. No 2.x versions are installed. 3.0.0 is installed. An error message is printed.
+
+Minor version roll-forward has one side-effect that may affect end users. Consider the following scenario:
+
+- 2.0.4 is required. No 2.0.* versions are installed. 2.2.2 is installed. 2.2.2 is used.
+- 2.0.5 is later installed. 2.0.5 will be used for subsequent application launches, not 2.2.2.
+- It's possible that 2.0.5 and 2.2.2 might behave differently, particularly for scenarios like serializing binary data.
+
+All NuGet dependencies are resolved as part of the publish operation and reside as assemblies (.dlls) in a flat directory structure. These assemblies should run on .NET Core 2.1 due to its compatibility promise for existing applications and binaries.
+
+Developers should remember the following rules:
```

Comment:
I'm not sure it's helpful to restate the logic here.  I would consider deleting form here through the end of the section.

Reply:
removed as well.